automated-comments/go/two-fer/strings_join_used_for_concatenation.md

```diff
@@ -1,2 +1,3 @@
-Don't use `strings.Join` if the data is not of type `[]string`.
-Use `fmt.Sprintf` instead. It's a very powerful way of building up strings from multiple pieces.
+`strings.Join` is only normally used if the data is of type `[]string`.
+In other situations, it is normal to use `fmt.Sprintf`, which is a very powerful way of building up strings from multiple pieces.
```

Comment:
Maybe this is me being Dutch, but "normal" felt a bit odd to me. It felt a bit like saying that not doing it is abnormal. Maybe "customary" is better? Feel free to ignore if this really _is_ me being Dutch :D

```suggestion
In other situations, it is customary to use `fmt.Sprintf`, which is a very powerful way of building up strings from multiple pieces.
```

Reply:
'common' ? 'more common' ?

Reply:
It depends on whether it's normal or common! :) If nearly everyone does it, then I'd say normal (e.g. it's an idiom or close to it). If it's common (eg. 60% of people do it) then I'd say "common" but then is it in an idiom and if not, should we not say it's common or normal, but just state it as an alternative?

Reply:
Thanks for the explanation. I'll then leave it to the go-people to determine if it is normal or common :)

Reply:
It's not like everybody does it, no. So `common` seems correct according to the definition @iHiD gave.

Reply:
Perhaps "more common" is the best compromise here. I often like to say that certain things in Go are "conventional", too—and I mean that in a good way! Gophers love being conventional.

Reply:
```suggestion
In other situations, it is more common to use `fmt.Sprintf`, which is a very powerful way of building up strings from multiple pieces.
```

Reply:
It is "more common" than what? Perhaps "it is common" is better here.